<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaPorFecha</class>
 <widget class="QWidget" name="ventanaPorFecha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consultar por fecha</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Reporte por fecha</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QCalendarWidget" name="calendarPorFecha">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="datosContenedor" native="true">
        <widget class="QLabel" name="desdeLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>50</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Desde
</string>
         </property>
        </widget>
        <widget class="QLabel" name="hastaLabel">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>90</y>
           <width>71</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Hasta
</string>
         </property>
        </widget>
        <widget class="QPushButton" name="setBtn">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>190</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
        <widget class="QComboBox" name="des_hasComboBox">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Desde</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Hasta</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="cancelarBtn">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="confirmarBtn">
        <property name="text">
         <string>Confirmar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
